src/Sarif.Viewer.VisualStudio.Core/SarifCommandPackage.vsct

```diff
@@ -159,14 +159,15 @@
             <ButtonText>Suppress result in SARIF file</ButtonText>
           </Strings>
       </Button>
-      <Button guid="guidSarifErrorListCmdSet" id="cmdidIFixedThisCommand" priority="0x0100" type="Button">
+      <!--Can be used to hide an error list item until the end of the VS session.--> 
```

Comment:
><!-- [](http://example.com/codeflow?start=6&length=4)

On second thought, I think it might be better to just keep the item hidden. To do that, comment out line 238 in .\src\Sarif.Viewer.VisualStudio.Core\ErrorList\ErrorListCommand.cs